src/stream/src/task/stream_manager.rs

```diff
@@ -587,7 +577,9 @@ impl LocalStreamManagerCore {
             let handle = {
                 let actor = async move {
                     // unwrap the actor result to panic on error
-                    actor.run().await.expect("actor failed");
+                    let _ = actor.run().await.inspect_err(|_e| {
+                        // TODO: check error type and panic if it's unexpected.
```

Comment:
How about simply logging the error for now?

Reply:
And also make this line a `bail!`?

https://github.com/risingwavelabs/risingwave/blob/be874bbe0520fc5a4966894b7e357bdd451fd7fe/src/stream/src/executor/actor.rs#L140